examples/tcp.rs

```diff
@@ -0,0 +1,7 @@
+//! This module provides examples for the TCP api.
+
+extern crate laminar;
+
+use self::laminar::net::{UdpSocket, TcpClient, TcpServer, TcpSocketState, SocketAddr};
+
+// TODO
```

Comment:
If we are gonna leave this TODO here, lets remove this file, and open an issue, instead.